service/src/main/java/com/ge/predix/acs/privilege/management/dao/TitanMigrationManager.java

```diff
@@ -0,0 +1,66 @@
+package com.ge.predix.acs.privilege.management.dao;
+
+import java.util.List;
+
+import javax.annotation.PostConstruct;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile("titan")
+public final class TitanMigrationManager {
+    private static final Logger LOGGER = LoggerFactory.getLogger(TitanMigrationManager.class);
+
+    @Autowired
+    @Qualifier("resourceRepositoryProxy")
+    private ResourceRepositoryProxy resourceProxy;
+
+    @Autowired
+    @Qualifier("subjectRepositoryProxy")
+    private SubjectRepositoryProxy subjectProxy;
+
+    public TitanMigrationManager(final ResourceRepositoryProxy resourceProxy,
+            final SubjectRepositoryProxy subjectProxy) {
+        this.resourceProxy = resourceProxy;
+        this.subjectProxy = subjectProxy;
+    }
+
+    public TitanMigrationManager() {
+
+    }
+
+    @PostConstruct
+    public void doMigration() {
+        // TODO: Return value should reflect status of migration?
+        // Success, failure, interrupted etc ?
```

Comment:
Please remove. Any TODOs should be captured in stories.  In this case, I dont see a need for a return code.